include/seqan3/alignment/pairwise/detail/policy_optimum_tracker.hpp

```diff
@@ -83,16 +86,18 @@ class policy_optimum_tracker
      * \tparam cell_t The cell type of the alignment matrix; must have a member function `optimal_score()`.
      *
      * \param[in] cell The current cell to be tracked.
+     * \param[in] coordinate The matrix coordinate of the current cell.
      *
      * \returns The forwarded cell.
      *
+     *
      * \details
      *
      * A call to this function only tracks the optimal score of the given cell if the configuration of the alignment
      * algorithm requires it, for example when a local alignment shall be computed.
      */
     template <typename cell_t>
-    decltype(auto) track_cell(cell_t && cell) noexcept
+    decltype(auto) track_cell(cell_t && cell, matrix_coordinate SEQAN3_DOXYGEN_ONLY(coordinate)) noexcept
```

Comment:
This looks a bit weird, if you don't really need them, but maybe we need them in a different implementation :)

Reply:
I wanted to have the interface already inside of the algorithm. Of course it was done on purpose since it will be implemented soonish